Mathlib/NumberTheory/EulerProduct/ExpLog.lean

```diff
@@ -0,0 +1,57 @@
+/-
+Copyright (c) 2024 Michael Stoll. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Michael Stoll
+-/
+import Mathlib.Analysis.SpecialFunctions.Complex.LogDeriv
+import Mathlib.NumberTheory.EulerProduct.Basic
+
+/-!
+# Logarithms of Euler Products
+
+We consider `f : ℕ →*₀ ℂ` and show that `exp (∑ p in Primes, log (1 - f p)⁻¹) = ∑ n : ℕ, f n`
+under suitable conditions on `f`. This can be seen as a logarithmic version of the
+Euler product for `f`.
+-/
+
+open Complex
+
+attribute [local fun_prop] DifferentiableAt.clog
```

Comment:
maybe add a comment explaining why this is only done locally?

Reply:
Done.